core/src/main/java/jenkins/telemetry/impl/SecuritySystemProperties.java

```diff
@@ -97,7 +98,12 @@ public JSONObject createContent() {
         putStringInfo(security, "hudson.security.HudsonPrivateSecurityRealm.ID_REGEX");
 
         Map<String, Object> info = new TreeMap<>();
-        info.put("core", Jenkins.getVersion().toString());
+        VersionNumber jenkinsVersion = Jenkins.getVersion();
+        if (jenkinsVersion != null) {
+            info.put("core", jenkinsVersion.toString());
+        } else {
+            info.put("core", "UNKNOWN");
+        }
```

Comment:
Maybe something like info.put("core", jenkinsVersion =! null ? jenkinsVersion.toString() : "UNKNOWN") so that it's less cumbersome?